test/TEST-85-NETWORK/meson.build

```diff
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+name = fs.name(meson.current_source_dir())
+unit = configure_file(
+        input : files('../test.service.in'),
+        output : '@0@.service'.format(name),
+        configuration : integration_test_template['configuration'] + {
+                'command' : '@0@ --no-journal'.format(testdata_dir / 'test-network/systemd-networkd-tests.py')
+        },
+)
+
+foreach testcase : [
+        'NetworkctlTests',
+        'NetworkdMatchTests',
+        'WaitOnlineTests',
+        'NetworkdNetDevTests',
+        'NetworkdL2TPTests',
+        'NetworkdNetworkTests',
+        'NetworkdTCTests',
+        'NetworkdStateFileTests',
+        'NetworkdBondTests',
+        'NetworkdBridgeTests',
+        'NetworkdSRIOVTests',
+        'NetworkdLLDPTests',
+        'NetworkdRATests',
+        'NetworkdDHCPServerTests',
+        'NetworkdDHCPServerRelayAgentTests',
+        'NetworkdDHCPClientTests',
+        'NetworkdDHCPPDTests',
+        'NetworkdIPv6PrefixTests',
+        'NetworkdMTUTests',
```

Comment:
Is it possible to get the list dynamically? Otherwise, newly added tests are easily forgotten.

Reply:
I tried but couldn't find a way to get it working. If you can figure out how to get the tests dynamically we can update this to use that.

Reply:
How about to call simple command something like the following?
```
sed -ne '/^class .*Tests/ { s/^class *//; s/(.*$//; p}' systemd-networkd-tests.py
```
But, it is OK to done later in another PR.